CMSIS/DSP/Examples/ARM/arm_nn_examples/gru/arm_nnexamples_gru.cpp

```diff
@@ -1,29 +1,40 @@
-/*
- * Copyright (C) 2010-2017 ARM Limited or its affiliates. All rights reserved.
- *
- * SPDX-License-Identifier: Apache-2.0
- *
- * Licensed under the Apache License, Version 2.0 (the License); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an AS IS BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 /* ----------------------------------------------------------------------
- * Project:      CMSIS-NN
- * Title:        arm_nnexamples_gru.cpp
- * Description:	 Gated Recurrent Unit Example
- *
- * Target Processor: Cortex-M4 and Cortex-M7 cores
- *
- * -------------------------------------------------------------------- */
+* Copyright (C) 2010-2017 ARM Limited. All rights reserved.
```

Comment:
Please assure all copyright statements are up to date, i.e. this one should become 
> Copyright (C) 2010-2018 Arm Limited. All rights reserved.